scripts/status_probe.py

```diff
@@ -35,9 +39,24 @@
 )
 args = parser.parse_args()
 
-resp = urllib.request.urlopen(args.status_url)
-content = resp.read()
-app_status = json.loads(content.decode("utf8"))
+if args.status_url:
+    resp = requests.get(args.status_url)
+    if resp.status_code != 200:
+        raise Exception(f"Server did not responde with 200! ({resp.status_code})")
+
+    app_status = resp.json()
+else:
+    if args.filename == "-":
+        jfile = sys.stdin
+    else:
+        jfile = open(args.filename, "rt")
+
+    content = jfile.read()
+    if jfile != sys.stdin:
+        jfile.close()
+    app_status = json.loads(content)
```

Comment:
`open()` and `read()` and `json.loads()` can all raise exceptions.

Do you want to catch any of them?

Reply:
I should at least log 'em.

Reply:
Very logging enabled.